src/test/java/step_definitions/Hooks.java

```diff
@@ -0,0 +1,20 @@
+package step_definitions;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import utils.DriverUtils;
+
+public class Hooks {
+    @Before
+    public void setUp(){
+        DriverUtils.createDriver("http://18.116.88.132:8080/bank/login");
+    }
+    @After
+    public void cleanUp() throws InterruptedException {
+        Thread.sleep(3000);
```

Comment:
Please change this from 3000 ms to 5000 ms.

Reply:
done!